pkg/apis/controlplane/v1beta2/zz_generated.conversion.go

```diff
@@ -489,17 +489,7 @@ func RegisterConversions(s *runtime.Scheme) error {
 
 func autoConvert_v1beta2_AddressGroup_To_controlplane_AddressGroup(in *AddressGroup, out *controlplane.AddressGroup, s conversion.Scope) error {
 	out.ObjectMeta = in.ObjectMeta
-	if in.GroupMembers != nil {
```

Comment:
Is this caused by the move of `Node *NodeReference`?

Reply:
I think so, it's generated code, surprised to see this shortened conversion.